extractor/image/image.go

```diff
@@ -69,6 +73,16 @@ func NewImage(ctx context.Context, image Reference, transports []string, option
 		auth = GetToken(ctx, domain, option)
 	}
 
+	// Receive an image content from stdin
+	var tmpImageFileName string
+	if image.IsFile && image.Name == "-" && !terminal.IsTerminal(0) {
```

Comment:
Is there a way to set something up so that `!terminal.IsTerminal(0)` can evaluate to `true` in a test? If so we should add a test for it.